src/main/java/no/nav/familie/integrasjoner/aktør/internal/AktørregisterClient.java

```diff
@@ -60,9 +59,9 @@
         HttpRequest request = HttpRequest.newBuilder()
                 .uri(uri)
                 .header(ACCEPT, "application/json")
-                .header(NAV_PERSONIDENTER, personIdent)
-                .header(NAV_CONSUMER_ID, consumer)
-                .header(NAV_CALL_ID, MDCOperations.getCallId())
+                .header(NAV_PERSONIDENT.asString(), personIdent)
```

Comment:
Trenger vi asString()?

Reply:
Uten asString() vil gi NAV_CALL_ID i stedet for Nav-Call-Id, men godt spm!